backend/data/helpers/userModel.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../dbConfig.js');
+
+module.exports = {
+  // get - with and without id
+  get: function(id) {
+    let query = db('users');
```

Comment:
Another way of doing this
```suggestion
    return id ? db('users').where('user_id', id).first() : db('users')
```

Reply:
I like that! Thanks!